### PR TITLE
Replace Gitter link with Discord/

### LINKED ISSUE #92: replace Gitter link with discord
CleanShot 2023-09-26 at 06 56 31@2x

### PR DESCRIPTION
**Pull request by wetted**
closes #92

### CODE REVIEW

**Comment by sdelamo** [comment]
I think this is blocked because of https://github.com/micronaut-projects/micronaut-starter-ui/issues/94

**Comment by timyates** [comment]
I think adding this makes the Gradle task pass

```diff
diff --git a/build.gradle b/build.gradle
index ce28f3f..c0a11a0 100644
--- a/build.gradle
+++ b/build.gradle
@@ -18,6 +18,7 @@ node {
 task buildStarter(type: NpmTask) {
     dependsOn npmInstall
     workingDir = file("${rootProject.projectDir}/app/launch")
+    environment = [NODE_OPTIONS: '--openssl-legacy-provider']
     args = ['run', 'build']
 }
```

Which may unblock this, but we really should do #94 

I'm not sure why this is required either... Locally to run the apps, I seem to need to use

```
❯ nvm use v12.22.10
Now using node v12.22.10 (npm v6.14.16)
```

But the change being required seems to imply that the gradle build is using Node 17

I have no idea what's going on

File: app/launch/src/components/Links/DiscordLink.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      >
+        <path d="M689.43 349a422.21 422.21 0 0 0-104.22-32.32 1.58 1.58 0 0 0-1.68.79 294.11 294.11 0 0 0-13 26.66 389.78 389.78 0 0 0-117.05 0 269.75 269.75 0 0 0-13.18-26.66 1.64 1.64 0 0 0-1.68-.79A421 421 0 0 0 334.44 349a1.49 1.49 0 0 0-.69.59c-66.37 99.17-84.55 195.9-75.63 291.41a1.76 1.76 0 0 0 .67 1.2 424.58 424.58 0 0 0 127.85 64.63 1.66 1.66 0 0 0 1.8-.59 303.45 303.45 0 0 0 26.15-42.54 1.62 1.62 0 0 0-.89-2.25 279.6 279.6 0 0 1-39.94-19 1.64 1.64 0 0 1-.16-2.72c2.68-2 5.37-4.1 7.93-6.22a1.58 1.58 0 0 1 1.65-.22c83.79 38.26 174.51 38.26 257.31 0a1.58 1.58 0 0 1 1.68.2c2.56 2.11 5.25 4.23 8 6.24a1.64 1.64 0 0 1-.14 2.72 262.37 262.37 0 0 1-40 19 1.63 1.63 0 0 0-.87 2.28 340.72 340.72 0 0 0 26.13 42.52 1.62 1.62 0 0 0 1.8.61 423.17 423.17 0 0 0 128-64.63 1.64 1.64 0 0 0 .67-1.18c10.68-110.44-17.88-206.38-75.7-291.42a1.3 1.3 0 0 0-.63-.63zM427.09 582.85c-25.23 0-46-23.16-46-51.6s20.38-51.6 46-51.6c25.83 0 46.42 23.36 46 51.6.02 28.44-20.37 51.6-46 51.6zm170.13 0c-25.23 0-46-23.16-46-51.6s20.38-51.6 46-51.6c25.83 0 46.42 23.36 46 51.6.01 28.44-20.17 51.6-46 51.6z"
+              style="fill:#fff"/>
+      </svg>
+    </a>
+  )
+}
+export default DiscordLink
```
Comment (timyates) [inline app/launch/src/components/Links/DiscordLink.js:32]:
```suggestion
export const DiscordLink = ({ className, theme }) => {
  const backgroundColor = theme === 'dark' ? 'var(--theme-light)' : 'var(--theme-dark)'
  const fill = theme === 'dark' ? '#000' : '#fff'

  return (
    <a
      href="https://discord.com/channels/1121511613250412714/1122324729756385300"
      aria-label="Micronaut Discord Channel"
      title="Micronaut Discord Channel"
      target="_blank"
      rel="noopener noreferrer"
      style={{
        backgroundColor,
      }}
      className={className}
    >
      <svg
        aria-hidden="true"
        focusable="false"
        viewBox="-50 -28.5 356 256"
      >
        <path d="M216.856339,16.5966031 C200.285002,8.84328665 182.566144,3.2084988 164.041564,0 C161.766523,4.11318106 159.108624,9.64549908 157.276099,14.0464379 C137.583995,11.0849896 118.072967,11.0849896 98.7430163,14.0464379 C96.9108417,9.64549908 94.1925838,4.11318106 91.8971895,0 C73.3526068,3.2084988 55.6133949,8.86399117 39.0420583,16.6376612 C5.61752293,67.146514 -3.4433191,116.400813 1.08711069,164.955721 C23.2560196,181.510915 44.7403634,191.567697 65.8621325,198.148576 C71.0772151,190.971126 75.7283628,183.341335 79.7352139,175.300261 C72.104019,172.400575 64.7949724,168.822202 57.8887866,164.667963 C59.7209612,163.310589 61.5131304,161.891452 63.2445898,160.431257 C105.36741,180.133187 151.134928,180.133187 192.754523,160.431257 C194.506336,161.891452 196.298154,163.310589 198.110326,164.667963 C191.183787,168.842556 183.854737,172.420929 176.223542,175.320965 C180.230393,183.341335 184.861538,190.991831 190.096624,198.16893 C211.238746,191.588051 232.743023,181.531619 254.911949,164.955721 C260.227747,108.668201 245.831087,59.8662432 216.856339,16.5966031 Z M85.4738752,135.09489 C72.8290281,135.09489 62.4592217,123.290155 62.4592217,108.914901 C62.4592217,94.5396472 72.607595,82.7145587 85.4738752,82.7145587 C98.3405064,82.7145587 108.709962,94.5189427 108.488529,108.914901 C108.508531,123.290155 98.3405064,135.09489 85.4738752,135.09489 Z M170.525237,135.09489 C157.88039,135.09489 147.510584,123.290155 147.510584,108.914901 C147.510584,94.5396472 157.658606,82.7145587 170.525237,82.7145587 C183.391518,82.7145587 193.761324,94.5189427 193.539891,108.914901 C193.539891,123.290155 183.391518,135.09489 170.525237,135.09489 Z"
              style={{ fill:fill }}
        />
      </svg>
    </a>
  )
}
export default DiscordLink
```

That gives:

image

For light mode and

image

For dark mode

**Review by timyates — CHANGES_REQUESTED** [review]

**Comment by timyates** [comment]
> I have no idea what's going on

[I've raised a PR to update to the latest node plugin and gradle version (and update some of the packages)](https://github.com/micronaut-projects/micronaut-starter-ui/pull/95)

**Comment by timyates** [comment]
To run this PR locally and see the result;

1. Run micronaut-starter locally with `CORS_ALLOWED_ORIGIN=http://localhost:3000 ./gradlew starter-web-netty:run`
2. In another terminal, in starter-ui run

```
cd dev-proxy-server && \
    rm -rf package-lock.json node_modules && \
    npm install && \
    npm run start:starter
```

3. In yet another terminal, run:

```
cd app/launch && \
    rm -rf package-lock.json node_modules && \
    npm install && \
    NODE_OPTIONS='--openssl-legacy-provider' npm run start:local
```

Your browser should open to http://localhost:3000/launch

**Comment by sdelamo** [comment]
It looks black to me 
CleanShot 2023-10-03 at 13 34 04@2x

**Comment by timyates** [comment]
[With the updated discord.js](https://github.com/micronaut-projects/micronaut-starter-ui/pull/93#pullrequestreview-1650369760)?

Which browser is that in?

**Comment by timyates** [comment]
Ahh yes... it's black in safari 🫠 looking...

**Comment by timyates** [comment]
@sdelamo can you try now?

**Review by timyates — APPROVED** [review]

**Review by sdelamo — CHANGES_REQUESTED** [review]
It displays a black circle

**Review by sdelamo — APPROVED** [review]
Thanks @timyates

**Pull request merged**